cdc/log.cc

```diff
@@ -1320,6 +1291,28 @@ class transformer final : public change_processor {
         return _touched_parts;
     }
 
+    bytes_opt get_col_from_row_state(const cell_map* state, const column_definition& cdef) const {
+        if (state) {
+            if (auto it = state->find(&cdef); it != state->end()) {
+                return it->second;
+            }
+        }
+        return std::nullopt;
+    }
+
+    cell_map* get_current_row_state(const clustering_key* ck) {
+        if (!ck) {
```

Comment:
Wouldn't it make sense to split those two cases apart into separated methods?
I have an impression that get_current_row_state with null and get_current_row_state with not null are called in different places so it will be not only more efficient but also more clear to have get_current_static_row_state() and get_current_ck_row_state(const clustering_key* ck).

Reply:
I agree. Done.